### ISSUE TITLE
About the model performance

### ISSUE THREAD

**Issue by shanjiayao** [Jabb0/FastFlow3D#7]
Hello @Jabb0 , thanks for your implementation!

I have some questions about the performance of this paper. After your model is trained, can the test results reach the accuracy in the paper? And could you share the test results?

**Comment by Jabb0** [comment]
The results in the readme are done with BatchNorm as in the original paper.
The default is BatchNorm.
But GroupNorm can be used instead via an command line option.

Yes the imbalance is likely a cause. That is why there is a downweight of the background class in the loss function.
In general points that are slower have less error (compare moving Vs. All in the table), indicating some bias towards background points (which have 0 flow).
If you have an idea for dealing with the imbalances try that.

Also the cyclist class has very few points. First bicycles are small and second (I guess) not common in the US.

However, a large portion of points belonging to vehicles are up to 1 m/s correct.

**Comment by shanjiayao** [comment]
Okay, thanks for your reply. I will continue to try to improve the performance of the model, and I will contact you as soon as there is progress. Thank you very much!
